libs/cohere/README.md

```diff
@@ -12,6 +12,21 @@ pip install langchain-cohere
 
 - Get a [Cohere API key](https://cohere.com/) and set it as an environment variable (`COHERE_API_KEY`)
 
+## Migration from langchain-community
+
+Cohere's integrations used to be part of the `langchain-community` package, but since version the integration in `langchain-community` has been deprecated in favour `langchain-cohere`.
```

Comment:
```suggestion
Cohere's integrations used to be part of the `langchain-community` package, but since version <TBA?> the integration in `langchain-community` has been deprecated in favour `langchain-cohere`.
```

Reply:
Done, thank you!